Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -618,6 +620,22 @@
                         sortable: false,
                         filterable: false
                     },
+                    {
+                        field: "LastChangedName", title: "Sidst redigeret: Bruger", width: 150,
+                        persistId: "lastchangedname", // DON'T YOU DARE RENAME!
+                        template: dataItem => dataItem && dataItem.LastChangedByUser ? `${dataItem.LastChangedByUser.Name} ${dataItem.LastChangedByUser.LastName}` : "",
+                        hidden: true,
+                        sortable: false,
+                        filterable: false
+                    },
+                    {
+                        field: "LastChanged", title: "Sidste redigeret: Dato", width: 150,
+                        persistId: "lastchanged", // DON'T YOU DARE RENAME!
+                        template: dataItem => dataItem?.LastChanged ? dataItem.LastChanged.toString() : "",
+                        hidden: true,
+                        sortable: true,
+                        filterable: false
+                    },
                     {
                         field: "DataProcessingRegistrationsConcluded", title: "Databehandleraftale", width: 150,
```

Comment:
@a-naskret You forgot to add it to the other overview as well (there are two overview pages for it-contracts)

![image](https://user-images.githubusercontent.com/13116889/171377703-d2b42348-2204-4dca-91e7-bc9eae98ee23.png)